code/style.css

```diff
@@ -1,4 +1,412 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background:#f2f2f2;
+  color:black;
 }
+
+body {
+  width: 100%;
+  margin: auto;
+  font-family: 'Inter', sans-serif;
+}
+
+/* HEADER */
+
+.header {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  padding: 17px;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.logo-container {
+  position: relative;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  overflow: hidden;
+  z-index: 3;
+}
+
+.logo-video {
+  display: none;
+}
+
+.logo-container:hover .logo-video {
+  display: inline-block;
+  width: 170px;
+  height: 170px;
+  object-fit: cover;
+  z-index: 0;
+  position: absolute;
+}
+
+.logo {
+  font-family: 'Jacques Francois Shadow', cursive;
+  font-size: 30px;
+  z-index: 2;
+  padding: 5px;
+}
+
+.logo:hover {
+  color: white;
+  text-shadow: 2px 1px 1px black;
+}
+
+@keyframes nav-animation {
+  0%   {background-color: white; color:black;}
+  25%  {background-color: #FF70A6; color:white;}
+  50%  {background-color: #7765E3; color:white;}
+  100% {background-color: #FCF300; color:black;}
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu {
+  text-decoration: none;
+  color:black;
+  font-size: 12px;
+  font-weight: 800;
+  padding: 3px;
+}
+
+.menu:hover {
+  box-shadow: 4px 4px black;
+  animation-name: nav-animation;
+  animation-duration: 3s;
+  animation-iteration-count: infinite;
```

Comment:
you can put this on one line
See more about [this here](https://css-tricks.com/almanac/properties/a/animation/)